### PR TITLE
54580_public_activity_gem

### PR DESCRIPTION
**Pull request by vietbq**

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<h1>Users#index</h1>
```
Comment (ggbond2605) [inline app/views/users/index.html.erb]: I18n

File: app/views/users/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+<p>Find me in app/views/users/index.html.erb</p>
```
Comment (ggbond2605) [inline app/views/users/index.html.erb]: I18N hoac bo di

File: app/models/comment.rb [hunk 3]
```diff
@@ -1,4 +1,7 @@
 class Comment < ActiveRecord::Base
+  include PublicActivity::Model
+  tracked owner: ->(controller, model) { controller && controller.current_user }
```
Comment (hellovietnam93) [inline app/models/comment.rb]: convention

**Comment by hellovietnam93** [comment]
OK

**Pull request merged**